command/node_status.go

```diff
@@ -422,7 +422,7 @@ func (c *NodeStatusCommand) outputNodeEvent(events []*api.NodeEvent) {
 	}
 
 	for i, event := range events {
-		timestamp := formatUnixNanoTime(event.Timestamp)
+		timestamp := formatTime(time.Unix(event.Timestamp, 0))
```

Comment:
TaskEvents (and I _think_ every other timestamp field) use nanoseconds since epoch, so perhaps switch the struct definition instead of the formatting?